client/views/windows/settings.tpl

```diff
@@ -14,6 +14,10 @@
 	</div>
 
 	<div class="row">
+		<div class="col-sm-12">
+			<button type="button" class="btn" id="make-default-client">Open irc:// URLs in The Lounge</button>
```

Comment:
`Opent <code>irc://</code> and <code>ircs://</code> in...`? (Also, `with` instead of `in`? Don't trust me on that one)

Reply:
Would be pretty long to include `ircs://` in the button?

Reply:
Oooh I'm dumb, I didn't even pay attention that it wasn't a checkbox!
Yeah it absolutely would. In fact, remove `<code>` as well actually, it's pretty gross on buttons. We can re-think that when we improve buttons (v3.1? 😁).